build.rs

```diff
@@ -28,6 +28,22 @@ fn main() {
         println!("cargo:rustc-cfg=feature=\"mem\"");
     }
 
+    let target_os = env::var("CARGO_CFG_TARGET_OS").unwrap();
+    if target_os != "windows" {
```

Comment:
I think it's probably fine to just check the `target` local variable like the rest of this function